tester.go

```diff
@@ -18,7 +20,20 @@ type Tester struct {
 
 // newTester creates a Tester based on the TesterDefinition provided
 func newTester(env map[string]string, definition tester_definition.TesterDefinition) (Tester, error) {
-	context, err := tester_context.GetTesterContext(env, definition.ExecutableFileName)
+	submissionDir, ok := env["CODECRAFTERS_SUBMISSION_DIR"]
+	if !ok {
+		return Tester{}, fmt.Errorf("CODECRAFTERS_SUBMISSION_DIR env var not found")
+	}
+
+	// check for your_program.sh
+	_, err := os.Stat(path.Join(submissionDir, definition.ExecutableFileName))
+	var context tester_context.TesterContext
+	if os.IsExist(err) {
```

Comment:
@glokta1 How have we tested this? Shouldn't err be `nil` if the file exists?

Reply:
Let's add a test for this behaviour, like the one we have here: https://github.com/codecrafters-io/tester-utils/blob/f721587ae3e8653b9216e25ff2a541a9d2211aab/tester_test.go#L37